contentctl/objects/abstract_security_content_objects/detection_abstract.py

```diff
@@ -161,12 +161,26 @@ def search_obsersables_exist_validate(cls, v, values):
         # Found everything
         return v
 
-    @validator("tests")
+    @validator("tests", always=True)
     def tests_validate(cls, v, values):
-        if values.get("status","") == DetectionStatus.production.value and not v:
-            raise ValueError(
-                "At least one test is REQUIRED for production detection: " + values["name"]
-            )
+        #Only production analytics require tests
+        if values.get("status","") != DetectionStatus.production.value:
+            return v
+        
+        #Only certain types of analytics must have tests
+        if values.get("type","") not in set([AnalyticsType.Anomaly.value, AnalyticsType.Hunting.value, AnalyticsType.TTP.value]):
```

Comment:
nit: I'd personally rather see this as a check against the list of the types which don't need tests, rather then the analytics which DO need tests. The subtle difference would be that any new types added would require tests by default and need to be added to this list as an exception

Reply:
I agree, the code has been updated to reflect this.